test/presubmit-tests-with-pipeline-deployment.sh

```diff
@@ -61,6 +64,7 @@ while [ "$1" != "" ]; do
 done
 
 # Variables
+GCR_IMAGE_BASE_DIR=gcr.io/${PROJECT}/${PULL_PULL_SHA}
```

Comment:
AFAIK, GCR buckets are different from projects. This should be configured separately.

Reply:
I got the impression from this doc: https://cloud.google.com/container-registry/docs/quickstart#tag_the_image_with_a_registry_name
It seems to suggest gcr.io/[project-id]/... is the format.
Please correct me if I didn't understand something correctly.

Theoretically, this script can use a different registry, but we don't have such a use case yet. Therefore, I kept it not configurable.